src/main/java/com/example/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -38,4 +38,16 @@ public List<BookDto> findAll() {
                 .map(bookMapper::toDto)
                 .toList();
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto requestDto) {
+        Book book = bookMapper.toModel(requestDto);
+        book.setId(id);
```

Comment:
Maybe we need an existing ID in the database check, but I am not sure

Reply:
Yes, we need it. (DB will throw an exception anyway, but by checking it here we can throw an exception with valuable message)